tests/dapr_tests.mk

```diff
@@ -90,7 +90,7 @@ DAPR_CONTAINER_LOG_PATH?=./dist/container_logs
 DAPR_TEST_LOG_PATH?=./dist/logs
 
 ifeq ($(DAPR_TEST_STATE_STORE),)
-DAPR_TEST_STATE_STORE=redis
+DAPR_TEST_STATE_STORE=postgres
```

Comment:
Let's keep this unrelated change in a different PR please.

Reply:
I still need to finalize this PR, but this is because redis doesn't support `ttlExpireTime`.